### ISSUE TITLE
SecureRandom `base58` doesn't work with Ruby 3.2.2 Rails 7.2.2

### ISSUE THREAD

**Issue by matheuss-kb4** [rails/rails#53850]
### Steps to reproduce

```ruby
rails new example
cd example
rails c
SecureRandom.base58
```

### Expected behavior

A random string is generated.

### Actual behavior

```
(example):1:in `<main>': wrong number of arguments (given 2, expected 0..1) (ArgumentError)
```

### System configuration
**Rails version**: 7.2.2

**Ruby version**: 3.2.2

**Comment by matheuss-kb4** [comment]
this is similar to https://github.com/rails/rails/issues/49260

**Comment by skipkayhil** [comment]
Fixed in https://github.com/rails/rails/commit/eff2f69d034273574302239b920242820490d727 but not yet released

**Comment by lucascaton** [comment]
Thanks for fixing it, @rafaelfranca 🙌

Will this be released in a future Rails version `7.2.2.2` or something?
